build.gradle

```diff
@@ -25,6 +25,20 @@ repositories {
 
 
 dependencies {
+    implementation 'org.projectlombok:lombok:1.18.16'
+    implementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'junit:junit:4.12'
+    implementation 'org.junit.jupiter:junit-jupiter:5.4.2'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'org.springframework.boot:spring-boot-starter-test'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    implementation 'junit:junit:4.12'
+    compile ('mysql:mysql-connector-java')
```

Comment:
Tutaj kod się powtarza. Najlepiej, żeby Hubert wpisał dependencies, żebyśmy potem nie mieli problemu z kodem

Reply:
to jest związane z testami i importami, które u mnie się wysypały